programmers2/k번째수.js

```diff
@@ -0,0 +1,10 @@
+function solution(array, commands) {
+    let answer = [];
+    for( let i = 0; i < commands.length; i++) {
+        let array2 = array.slice(commands[i][0] - 1, commands[i][1]);
+        array2.sort((a,b) => a - b);
+        let num = commands[i][2] - 1;
+        answer.push(array2[num]);
+    }
```

Comment:
1. for문 뒤의 공백 ,  for문 조건 (let 공백 없이가 맞는 컨벤션입니다.
2. 변수명을 좀더 신경써주세요 arr1 ,arr2 , num1 같은뒤에 숫자가 붙는 변수명은 정말 안좋습니다!
3. 가독성이 좀 더 좋아지도록 i, j, k 를 선언하였습니다.
4. sort((a, b)  컴마 뒤에도 공백을 붙여야 합니다.
commands[idx][0] - 1,
commands[idx][1], 
commands[idx][2] - 1
```suggestion
   const answer = [];
    for (let idx = 0; idx < commands.length; idx++) {
        const [i, j, k] = [
            commands[idx][0] - 1,
            commands[idx][1], 
            commands[idx][2] - 1
        ];
        
        let answers = array.slice(i, j).sort((a, b) => a - b);
        answer.push(answers[k]);
    }
```